projects/core/src/features-config/services/feature-config.service.ts

```diff
@@ -11,6 +11,11 @@ import {
   isFeatureLevel,
 } from '../utils/feature-config-utils';
 
+/**
+ * This service is not recommended for use.
+ *
+ * Please use `FeatureToggles` instead.
+ */
```

Comment:
`major:` `FeatureToggles` was meant to allow creating config factories depending on feature flags without causing a circular dependency issue - `FeatureConfigService` having `FeatureConfig` as dependency does so when injecting to a factory affecting the global `Config` object.
That said, FeatureConfigService is still ok to use, but not in config factories.